src/blocking/spi.rs

```diff
@@ -76,6 +137,29 @@ pub mod write {
         }
     }
 }
+/// Blocking write
+pub mod read {
+    /// Default implementation of `blocking::spi::Read<W>` for implementers
+    /// of `nonblocking::spi::FullDuplex<W>`
+    pub trait Default<W>: crate::nb::spi::FullDuplex<W> {}
+
+    impl<W, S> crate::blocking::spi::Read<W> for S
+    where
+        S: Default<W>,
+        W: core::default::Default,
+    {
+        type Error = S::Error;
+
+        fn read(&mut self, words: &mut [W]) -> Result<(), S::Error> {
+            for word in words.iter_mut() {
+                nb::block!(self.write(W::default()))?;
```

Comment:
The value written here should be documented.

Reply:
done